Automattic/jetpack
# Remove CTAs from AAG Dashboard

#### Changes proposed in this Pull Request:

* Remove the `HeaderNav` aka the Dashboard|Settings Button
* Remove the "Ask a Question" button from Support Section
* Change "View More Stats in WordPress.com" to a ExternalLink from a Button 

| Before | After |
| ------ | ----- |
| <img width="895" alt="Screen Shot 2021-12-17 at 10 16 09" src="https://user-images.githubusercontent.com/2810519/146590373-b0b575a8-d014-4e6a-8820-04554a25b756.png"> | <img width="895" alt="Screen Shot 2021-12-17 at 10 17 44" src="https://user-images.githubusercontent.com/2810519/146590325-3afabcb2-742c-4927-b2b4-1d5ad6c493b6.png"> |
| <img width="860" alt="Screen Shot 2021-12-17 at 10 15 52" src="https://user-images.githubusercontent.com/2810519/146590552-d04b7221-e931-4c67-b0ec-633c5489240b.png"> | <img width="860" alt="Screen Shot 2021-12-17 at 10 14 19" src="https://user-images.githubusercontent.com/2810519/146590535-247ed205-03fd-41c3-b3f4-f8c62c4efc71.png"> |
| <img width="832" alt="Screen Shot 2021-12-17 at 10 19 43" src="https://user-images.githubusercontent.com/2810519/146590248-d72b4e20-655c-468a-aa86-b977bd3c744c.png"> | <img width="832" alt="Screen Shot 2021-12-17 at 10 18 02" src="https://user-images.githubusercontent.com/2810519/146590441-d28b7a14-704e-4202-99f0-4440b57e1fe8.png"> | 

#### Jetpack product discussion



p1HpG7-dHF-p2#comment-51399



#### Does this pull request change what data or activity we track or use?



#### Testing instructions:






* Navigate to `/wp-admin/admin.php?page=jetpack#/dashboard` on branch
* Verify the changes described above are visible in the dashboard
